guided-meditations.md

```diff
@@ -416,28 +416,33 @@ dependency order and cycles are not permitted.
 There is a functional, though somewhat useless meditation:
 
 ```
-"Hello. How do you feel?"
-
-Choice
-  When "I am tried" Stop
-  When "I am confused"
-     Actions type = "sftp-symlink"
-     Stop
-  When "I'm looking for something more conventional"
-    Form
-      text = name With Label "What is your name?"
-    Then
-      Download "Hello, {name}! Does this meet your expectations"
-        To "example.txt" MimeType "text/plain"
+Start
```

Comment:
So does this PR imply guided meditation syntax changes? Do we need to rewrite our existing ones?

Reply:
No. The changes were already made and the documentation was stale.